spec/models/service_template_ansible_playbook_spec.rb

```diff
@@ -242,17 +241,18 @@
   end
 
   describe '#update_job_templates' do
-    let(:service_template) { prebuild_service_template(:job_template => false) }
+    let(:service_template) { prebuild_service_template(:job_template => true) }
 
     it 'does not update a job_template if the there is no playbook_id' do
-      expect(ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript)
-        .to receive(:update_in_provider_queue).never
-      service_template.send(:update_job_templates, 'blah', 'blah', catalog_item_options, user)
+      [:provision, :retirement, :reconfigure].each do |action|
+        catalog_item_options[:config_info][action].delete(:playbook_id) if catalog_item_options[:config_info][action]
```

Comment:
I think you can use #delete_path from more_core_extensions here adn drop the `if`.
```ruby
catalog_item_options.delete_path(:config_info, action, :playbook_id)
```